### PR TITLE
Added Simple Configuration API

### PR DESCRIPTION
**Pull request by KrzysztofLesniak**

### CODE REVIEW

**Review by pabloducato — COMMENTED** [review]
For me, ok, but I would like to hear @Calharad opinion

File: benchmark-backend/pom.xml [hunk 1]
```diff
@@ -61,6 +61,23 @@
...
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+            <version>2.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.modelmapper</groupId>
+            <artifactId>modelmapper</artifactId>
+            <version>2.3.8</version>
+        </dependency>
```
Comment (Calharad) [inline benchmark-backend/pom.xml:73]: we already have objectmapper from jackson library. Is it required to use another one?

**Review by Calharad — APPROVED** [review]
review ok, but check my one comment

File: benchmark-backend/pom.xml (hunk 1 above)
Comment (KrzysztofLesniak) [reply benchmark-backend/pom.xml]: Your're right, I already changed it in the next commit.

**Pull request merged**